src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13904,6 +13904,20 @@ namespace System.Runtime.InteropServices.Swift
         public unsafe SwiftSelf(void* value) { throw null; }
         public unsafe void* Value { get { throw null; } }
     }
+    [System.CLSCompliantAttribute(false)]
```

Comment:
```suggestion
```

This should be removed.

Reply:
I suspect that this is needed to make things compile

Reply:
his shouldn't be needed as `T` is CLS-compliant. In other cases, it is needed as `void*` is not CLS-compliant.